app/main.py

```diff
@@ -2,24 +1,65 @@
+from typing import Any, Type
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.max_amount = max_amount
+        self.min_amount = min_amount
+
+    def __set_name__(self, owner: Any, name: str) -> None:
+        self.protected_name = "_" + name
+
+    def __get__(self, instance: Any, owner: Any) -> Any:
+        return getattr(instance, self.protected_name)
+
+    def __set__(self, instance: Any, value: int) -> None:
+        if not isinstance(value, int):
+            raise TypeError("Value should be integer")
+        if value < self.min_amount or value > self.max_amount:
+            raise ValueError(f"Value should not be less than "
+                             f"{self.min_amount} and greater than "
+                             f"{self.max_amount}.")
+        setattr(instance, self.protected_name, value)
 
 
 class Visitor:
-   pass
+    def __init__(self, name: str, age: int, weight: int, height: int) -> None:
+        self.name = name
+        self.age = age
+        self.weight = weight
+        self.height = height
```

Comment:
The `Visitor` class attributes `age`, `weight`, and `height` are not using the `IntegerRange` descriptor. If the intention is to validate these attributes, consider using the `IntegerRange` descriptor here as well.